src/apps/invitations/services.py

```diff
@@ -106,6 +106,11 @@ async def send_respondent_invitation(
         # by user_id in this case
         invited_user = await UsersCRUD(self.session).get_user_or_none_by_email(email=schema.email)
         invited_user_id = invited_user.id if invited_user is not None else None
+        if invited_user_id:
+            await UserAppletAccessService(self.session, self._user.id, applet_id).unpin(
+                pinned_user_id=invited_user_id, pinned_subject_id=None
+            )
```

Comment:
We also have `unpin` when a participant accepts the invitation. Do we need it in both places?
If so, can you add a comment explaining why this `unpin` needs to happen here

Reply:
Both unpins are to fix database inconsistencies, during accepts is in case where user already getting error, but I prefer to fix this  possible inconsistency at first opportunity, so I add it during create of invitation 

it said, it can be removed from invitation creation